exasol_integration_test_docker_environment/test/test_hash_temp_dir.py

```diff
@@ -206,12 +216,161 @@ def test_directory_file_names_not_equal_to_dir_names(self):
                                     use_relative_paths=True,
                                     hash_directory_names=True,
                                     hash_file_names=False)
-        hash1_content_only = hasher_content_only.hash([self.test_dir1])
-        hash2_with_paths = hasher_with_paths.hash([self.test_dir2])
+        hash1_content_only = hasher_content_only.hash([simple_path_mapping(self.test_dir1)])
+        hash2_with_paths = hasher_with_paths.hash([simple_path_mapping(self.test_dir2)])
         ascii_hash1_content_only = base64.b32encode(hash1_content_only).decode("ASCII")
         ascii_hash2_with_paths = base64.b32encode(hash2_with_paths).decode("ASCII")
         self.assertNotEqual(ascii_hash1_content_only, ascii_hash2_with_paths)
 
+    def test_file_name_with_relative_path(self):
```

Comment:
You are modifying the test directories, this can interfere with other tests. We can't assume test execution order

Reply:
Fixed it. Create separate temporary directories.